unstructured_ingest/v2/examples/airtable.py

```diff
@@ -0,0 +1,44 @@
+import os
+from pathlib import Path
+
+from unstructured_ingest.v2.interfaces import ProcessorConfig
+from unstructured_ingest.v2.logger import logger
+from unstructured_ingest.v2.pipeline.pipeline import Pipeline
+from unstructured_ingest.v2.processes.chunker import ChunkerConfig
+from unstructured_ingest.v2.processes.connectors.airtable import (
+    CONNECTOR_TYPE,
+    AirtableAccessConfig,
+    AirtableConnectionConfig,
+    AirtableDownloaderConfig,
+    AirtableIndexerConfig,
+)
+from unstructured_ingest.v2.processes.connectors.local import (
+    LocalUploaderConfig,
+)
+from unstructured_ingest.v2.processes.embedder import EmbedderConfig
+from unstructured_ingest.v2.processes.partitioner import PartitionerConfig
+
+base_path = Path(__file__).parent.parent.parent.parent
+docs_path = base_path / "example-docs"
+work_dir = base_path / "tmp_ingest" / CONNECTOR_TYPE
+output_path = work_dir / "output"
+download_path = work_dir / "download"
+
+if __name__ == "__main__":
+    logger.info(f"Writing all content in: {work_dir.resolve()}")
+    Pipeline.from_configs(
+        context=ProcessorConfig(work_dir=str(work_dir.resolve()), verbose=True),
+        indexer_config=AirtableIndexerConfig(
+            list_of_paths=["app5YQxSfp220fWtm", "appJ43QmP8I17zu88"]
+        ),
+        downloader_config=AirtableDownloaderConfig(download_dir=download_path),
+        source_connection_config=AirtableConnectionConfig(
+            access_config=AirtableAccessConfig(
+                personal_access_token=os.getenv("AIRTABLE_PERSONAL_ACCESS_TOKEN")
```

Comment:
I understand for running this test locally, developer needs to register to Airtable, put there files from `test_e2e/expected-structured-output/airtable-diff/app5YQxSfp220fWtm` and set his PAT as this env. Is that right? 

Btw, I also see Airtable mentioned in e2e.yaml in GH action. So for that scenario, there is the code in `test_e2e/env_setup/airtable/create_scale_test_components.py` , and it's going to be run also for this v2 Airtable?

Reply:
Now that v2 is exposed, it's what will be used by the ingest tests.